code/modules/multiz/structures.dm

```diff
@@ -123,6 +123,37 @@
 	allowed_directions = UP|DOWN
 	icon_state = "ladder11"
 
+/obj/structure/ladder/mobile
+	allowed_directions = UP|DOWN
+	icon_state = "ladder11"
+	
+
+/obj/structure/ladder/mobile/verb/fold()
+	set name = "Fold Ladder"
+	set category = "Object"
+	set src in oview(1)
+
+	if(isobserver(usr) || usr.restrained() || !usr.Adjacent(src))
+		return
+
+	verbs -= /obj/structure/ladder/mobile/verb/fold
+	var /obj/item/weapon/ladder_mobile/R = new(src.loc)
+	src.transfer_fingerprints_to(R)
+	//src.Destroy()
+	
+	if(src.target_down == null)
+		src.target_up.Destroy()
```

Comment:
```
QDEL_NULL(src.target_up)
qdel(src)
```

You never call `.Destroy()` on an item directly. Unless you're Lohikar and fucking with the innards.